src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsMenuBarUI.java

```diff
@@ -146,17 +146,17 @@ public void actionPerformed(ActionEvent e) {
             if (menu != null) {
                 MenuSelectionManager msm =
                     MenuSelectionManager.defaultManager();
-                MenuElement[] path = new MenuElement[2];
-                path[0] = (MenuElement)menuBar;
-                path[1] = (MenuElement)menu;
-                if (!mnemonicShowHideFlag) {
-                    msm.setSelectedPath(path);
-                    WindowsLookAndFeel.setMnemonicHidden(false);
-                    mnemonicShowHideFlag = true;
-                } else {
+                MenuElement[] selectedPath = msm.getSelectedPath();
+                //MenuElement[] path = new MenuElement[2];
+                //path[0] = (MenuElement)menuBar;
+                //path[1] = (MenuElement)menu;
```

Comment:
Do not leave commented out code. The source control system is used to record the history.

If you want to keep the old way, you can still do it by replacing one line
```java
MenuElement[] path = {menuBar, menu};
```
inside `else` with these three lines.

Reply:
Oh! my bad. I pushed it by mistake. Will update in next iteration.